.gitignore

```diff
@@ -5,7 +5,7 @@ extra_vars.yml
 instructor*
 .vagrant/*
 ansible.cfg
-hosts
```

Comment:
Despite changing the name, I'd just assume leave this line in for backward compatibility issues.

Reply:
I don't mind leaving it in, but I also don't see the point.

Reply:
Not all inventory files are ini format, and we address this in the documentation here: https://docs.ansible.com/ansible/latest/intro_inventory.html

For consistency I'd like to keep hosts.

Reply:
I am confused, you like to keep *hosts* in *.gitignore*, even when it is not being used within this project ?